include/kernels/dslash_ndeg_twisted_clover_preconditioned.cuh

```diff
@@ -7,32 +7,32 @@
 
 namespace quda
 {
-
-  template <typename Float, int nColor, QudaReconstructType reconstruct_, bool dynamic_clover_>
```

Comment:
this template signature now looks different, I think (see `TwistedCloverArg`)

Reply:
Should be fixed now